HelpSource/Classes/Platform.schelp

```diff
@@ -144,6 +144,25 @@ files to be loaded on startup
 method:: loadStartupFiles
 (re)load startup files
 
+subsection:: Crossplatform system commands
+
+method:: killAll
+kill all processes as defined by cmdLineArgs.
+Typically, this is a process name or a list of names.
```

Comment:
these descriptions should use the `argument::` tag, and specify the expected type of the argument

Reply:
ok